crm/modules/user_meta/user_meta.inc.php

```diff
@@ -184,7 +184,8 @@ function user_meta_data ($opts = array()) {
         ";
     }
     if (!empty($opts['cid'])) {
-        $esc_cid = mysqli_real_escape_string($db_connect, $opts['cid']);
+        $cid = $opts['cid'][0];
+        $esc_cid = mysqli_real_escape_string($db_connect, $cid);
```

Comment:
nice job reducing the cognitive complexity here!

Reply:
TBH I preferred the 1 liner, PHP 7.4 didn't mind the 1 liner, PHP 8.1 hated the 1 liner :( :P